### ISSUE TITLE
Added de (Deutsch) translation

### ISSUE THREAD

**Issue by OffifialMITX** [vijai1996/screenrecorder#17]
I have translated your application into `de` (*Deutsch*). Here is the xml:
```xml
<resources>
    <string name="preference_resolution_title">Bildschirmauflösung</string>
    <string name="preference_resolution_summary">Wähle die Auflösung zum Aufnehmen mit</string>
    <string name="preference_fps_title">Bilder pro Sekunde</string>
    <string name="preference_fps_summary">Wähle Bilder/Sek. bei der Aufnahme mit</string>
    <string name="preference_bit_title">Bit-Rate</string>
    <string name="preference_bit_summary">Bit-Rate wählen</string>
    <string name="preference_audio_record_title">Audio aufnehmen</string>
    <string name="preference_audio_record_summary">Nimmt zusammen mit dem Bildschirmvideo Audio übers Mikrofon auf</string>
    <string name="preference_save_location_title">Speicherort</string>
    <string name="preference_filename_format_title">Dateinamensformat</string>
    <string name="preference_filename_prefix_title">Dateinamenspräfix</string>
    <string name="storage_permission_request_title">Erlaube Speicherberechtigung</string>
    <string name="storage_permission_request_summary">Schreib-Berechtigung auf dem Speicher ist erforderlich, um das aufgenommene Video zu speichern.</string>
    <string name="ok">OK</string>
    <string name="cancel">Abbrechen</string>
    <string name="screen_recording_permission_denied">Berechtigung zur Bildschirmaufnahme verwehrt</string>
    <string name="screen_recording_notification_title">Bildschirmaufnahme</string>
    <string name="screen_recording_notification_action_stop">Stopp</string>
    <string name="screen_recording_notification_action_pause">Pause</string>
    <string name="screen_recording_notification_action_resume">Fortsetzen</string>
    <string name="fatal_exception_message">ScreenRecorder: Fatale Ausnahmesituation eingetreten. Protokoll ist verschickt worden</string>
    <string name="about_activity_label">Über</string>
    <string name="snackbar_storage_permission_message">Speicherzugriff gewähren</string>
    <string name="snackbar_storage_permission_action_enable">AKTIVIEREN</string>
    <string name="menu_about_title">Über</string>
    <string name="about_credits">Danksagungen</string>
    <string name="about_libraries">Bibliotheken</string>
    <string name="about_opensource">Open Source</string>
    <string name="app_icon_credit_Niko">App-Symbole von %1$s\n\nWebsite: %2$s</string>
    <string name="directory_chooser_library_credit">Ordner-Wähler von %1$s\n\nGithub URL: %2$s\n\nLizenz: %3$s</string>
    <string name="opensource_info">Diese Anwendung ist quelloffen, und der Quellcode kann abgerufen werden unter\n\nGithub URL: %1$s\n\nLizenz: %2$s</string>
    <string name="fab_record_hint">Mitschnitt starten</string>
    <string name="recording_failed_toast">Bildschirmmitschnitt fehlgeschlagen.</string>
    <string name="screenrecording_already_active_toast">Bildschirmmitschnitt läuft schon</string>
    <string name="large_resolution_selected_toast">Die ausgewählte Auflösung ist größer als die Bildschirmauflösung. Wähle stattdessen %1$s Auflösung</string>
    <string name="shortcut_short_label">Aufnahme</string>
    <string name="shortcut_long_label">Schnellaufnahme</string>
    <string name="shortcut_disabled_message">Aufnahme abgeschaltet</string>
    <string name="screen_recording_started_toast">Bildschirmmitschnitt gestartet</string>
    <string name="screen_recording_stopped_toast">Bildschirmmitschnitt gestoppt</string>
    <string name="screen_recording_resumed_toast">Bildschirmmitschnitt fortgesetzt</string>
    <string name="screen_recording_paused_toast">Bildschirmaufnahme angehalten</string>
    <string name="fab_stop_hint">Mitschnitt stoppem</string>
    <string name="share_intent_title">Aufgenommenes teilen</string>
    <string name="share_intent_notification_title">Aufgenommenes Video teilen</string>
    <string name="share_intent_notification_content">Klicken, um das aufgenommene Video zu teilen</string>
    <string name="share_intent_notification_action_text">Teilen</string>
    <string name="preference_floating_control_title">Schwebende Bedienelemente nutzen</string>
    <string name="preference_floating_control_summary">Schwebende Bedienelemente ein-/ausschalten, die über jeden Bildschirm aufzeichen</string>
    <string name="tab_settings_title">Settings</string>
    <string name="tab_videos_title">Videos</string>
    <string name="video_list_permission_denied_message">Speicherzugriffsberechtigung verweigert</string>
    <string name="video_list_no_video_message">Keine Videos verfügbar</string>
</resources>

```

There were two strings which weren't translated.

Note: Some strings like "video settings", "save options" and so on aren't contained in the strings.xml

---

*Translation made with [Stringlate](https://github.com/LonamiWebs/Stringlate).*

**Comment by vijai1996** [comment]
Thanks for the note. I have fixed it. But, isn't German called Deutsch?

**Comment by OffifialMITX** [comment]
...yes, "Deutsch" is German in German but what do you mean?

**Comment by vijai1996** [comment]
Well, there is `de` already for German in the app. So I'm wondering what and where I went wrong

**Comment by OffifialMITX** [comment]
I can't understand what actually the problem is. I may have uploaded the translation twice accidentally, is this what you mean?

**Comment by vijai1996** [comment]
Ah! Seems like that. So I believe this issue is a duplicate of issue #15?

**Comment by OffifialMITX** [comment]
Yes, it's exactly what it is. May you open an issue for all languages, like "Translations"?

**Comment by OffifialMITX** [comment]
Here's an [update](https://gist.github.com/2eb0478672f1ad84bd80493df1b9c2b6)!

**Comment by vijai1996** [comment]
Not updating it for now since there is few more strings coming up within couple of days related to showing touch feedback in recorded video

**Comment by OffifialMITX** [comment]
Is that really necessary? I don't think so since you're able to set it in the developer's options.

(I like to see it anyway, not just during the recording)
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.

**Comment by vijai1996** [comment]
Well, the implementation is more or less similar and its there to ease out the need to enable it manually everytime. But, it will be a paid plugin and a separate app because the api has changed and its no more possible to enable it in dev options from app any more. But of course you can enable it manually without the plugin too.

Adding it based on feedback

**Comment by vijai1996** [comment]
@OffifialMITX I have updated the repo with all strings from my local repo. You may now update the German translation send me a PR. Thanks for your continuous effort and support

**Comment by OffifialMITX** [comment]
Is it ok if I'll just give you an URL to a gist? I don't know how to handle PRs.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.

**Comment by vijai1996** [comment]
Sure!

**Comment by OffifialMITX** [comment]
Really, you wanna make the plugin unfree? Though that means that your application will be greyed out  in F-Droid.

I mean, I don't have a problem with paid apps, just wanted to mention.

However, [here's](https://gist.github.com/b2f600ce0c9665d97583f44c09ae1518) the translation.

**Comment by vijai1996** [comment]
Well, the plug-in is paid yeah. But the main app itself doesn't rely on it. You can still build the app and it works just fine as it should. I don't see a reason why the app would be blocked on fdroid when the app still is foss

**Comment by OffifialMITX** [comment]
It won't be blocked but there's a label like "this app provides non-free addons" which makes the text in the same color like incompatible apps by standart (you can swich it of if you wish to).

What if you make a full-featured app for F-Droid, like some other apps (e.g. Secrecy or FTP server)?
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.

**Comment by OffifialMITX** [comment]
Another one [here](https://gist.github.com/7bbd42b60e6099bd2c8b5e2037a2d0a0), even if those are only two strings.

**Comment by OffifialMITX** [comment]
[Once again](https://gist.github.com/a4881437b526e86eebd41fc020145b5f)
.

**Comment by vijai1996** [comment]
Will be included in the next release. Thanks for the continuous support!

_Sent from my Huawei Nexus 6P using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by OffifialMITX** [comment]
No problem, feel free to contact me when you're about to release a new update so I can translate it before.

Am 4. Juni 2017 19:31:44 MESZ, schrieb Vijai Chander <notifications@github.com>:
>Will be included in the next release. Thanks for the continuous
>support!
>
>_Sent from my Huawei Nexus 6P using
>[FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/vijai1996/screenrecorder/issues/17#issuecomment-306054478

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.

**Comment by OffifialMITX** [comment]
New version of the current Beta branch https://gist.github.com/e26b230bbbf9876e9d9f620f97d99be8

**Comment by vijai1996** [comment]
@OffifialMITX Beta is not updated for long and I'm thinking of removing the branch sometime in future. Can you translate the master if possible? Sorry for the trouble.

**Comment by OffifialMITX** [comment]
Update for today's main branch! https://gist.github.com/c1ee08e0de78f8f4b7a450691cd9c47c

**Comment by vijai1996** [comment]
Thanks @OffifialMITX . I will merge this after I move the repo to gitlab (not happy with M$ acquisition) and get my broken Keyboard fixed
